app/build.gradle

```diff
@@ -50,7 +51,10 @@ android {
         }
 
         debug {
-            applicationIdSuffix ".debug"
+            //for getting the google play store to launch to the app application
+            //suffix needs to be blank
+            applicationIdSuffix ""
```

Comment:
@beastlyCoder Woul be better if the suffix exists in the debug version, It would work on release